backend.go

```diff
@@ -51,7 +51,12 @@ func newBackend(name string, m *Machine) (backend, error) {
 
 			b, backendErr := newBackend(backendName, m)
 			if backendErr != nil {
-				err = fmt.Errorf("%v, %v", err, backendErr)
+				/* Append the error to existing error */
+				if err != nil {
+					err = fmt.Errorf("%w, %w", err, backendErr)
```

Comment:
seems like two errors cannot be appended like this, hm

Reply:
that;s suprirsing; though mind you'd probably also want to put the backend name per error